lib/puppet/provider/ibm_pkg/imcl.rb

```diff
@@ -77,13 +89,19 @@ def self.find_installed_xml(user)
 
     installed_xml_path = nil
 
-    user_path = (user == 'root') ? '/var/ibm/' : "/home/#{user}/var/ibm/"
+    user_path = (user == 'root') ? '/var/ibm/' : "#{user_home(user)}/var/ibm/"
```

Comment:
Is it intended that this returns `/var/ibm` if the `user_home(user)` returns `nil`?